scripts/create_translation_file.php

```diff
@@ -44,6 +50,7 @@
 ####################
 
 $extensions      = explode( ',', $targetOption->value );
+$contexts        = explode( ',', $targetContexts->value );
```

Comment:
Could you rewrite this to:

$contexts = array();
if( $targetContexts->value )
{
    $contexts = explode( ',', $targetContexts->value );
}